_src/themes/mit/static/sass/_variables.scss

```diff
@@ -0,0 +1,77 @@
+// ----------------------------
+// MIT (Example file)
+// This file provides utility overrides for base/_utilities in the main platform compile.
+// It allows instutions to provide alternative values when displaying the application.
+// ----------------------------
+
+
+// ----------------------------
+// #COLORS
+// ----------------------------
+$red:                           rgba(163, 31, 52, 1.0);
+$dark-gray:                     rgba(138, 139, 140, 1.0);
+$light-gray:                    rgba(194, 192, 191, 1.0);
+
+$brand-primary:                 $red;
+$brand-secondary:               $dark-gray;
+$brand-accent:                  $light-gray;
+
+
+// ----------------------------
+// #TYPOGRAPHY
+// ----------------------------
+@include font-face('Open Sans', '../fonts/OpenSans/OpenSans-Light-webfont', 300, $file-formats: eot woff ttf svg);
```

Comment:
NIT: do we want to try to include a different typeface here to see the cascade working? If so, you could just import from Google Web Fonts to keep it simple and not add other asset files.